2022q4/wifi.adoc

```diff
@@ -0,0 +1,34 @@
+=== Wireless updates
+
+Links: +
+link:https://people.freebsd.org/~bz/wireless/[Bjoern's Wireless Work In Progress landing page] URL: link:https://people.freebsd.org/\~bz/wireless/[https://people.freebsd.org/~bz/wireless/] +
+
+Contact: Bjoern A. Zeeb <bz@FreeBSD.org>
+
+During the 4th quarter of 2022 not much work was publicly visible and admittedly slightly slow.
```

Comment:
> During the 4th quarter of 2022 

Instead: 

> During the quarter

Reply:
That makes no sense;  which of the 4 quarters?  During the last quarter of the year?

Reply:
It's a minor suggestion, for conciseness. 

> … During the last quarter of the year?

The period will be given in the introduction to the report, there's not yet an `intro.adoc` at <https://github.com/freebsd/freebsd-quarterly/tree/main/2022q4>. 

Produced from <https://github.com/freebsd/freebsd-quarterly/blob/main/2022q3/intro.adoc#readme>, a shot of the head of the report for the third quarter: 

![image](https://user-images.githubusercontent.com/192271/211212008-d461eb5f-dc3a-4539-b623-e0abdb0eecb1.png)

There's context – the period – in three places at beginning: 

1. the title bar (or tab)
2. the first heading
3. the first sentence

– and the title bar (or tab) context should remain visible throughout.

Reply:
Resolved by [`12e0e61` (#559)](https://github.com/freebsd/freebsd-quarterly/pull/559/commits/12e0e61a7af41468c093a6ae1fd82ee77fed7b48), thanks.